cinema/models.py

```diff
@@ -86,8 +104,9 @@ def clean(self):
             (self.seat, "seat", "count_seats_in_row"),
         ]:
             count_attrs = getattr(
-                self.movie_session.cinema_hall, cinema_hall_attr_name
-            )
+                self.movie_session.cinema_hall,
+                cinema_hall_attr_name)
```

Comment:
The `clean` method attempts to access attributes `count_rows` and `count_seats_in_row` on the `CinemaHall` model, but these attributes do not exist. You should use `rows` and `seats_in_row` instead, as these are the correct field names in the `CinemaHall` model.